README.adoc

```diff
@@ -241,15 +241,15 @@ The [hotspot=createMongo file=0]`createMongo()` producer method returns an insta
 
 The [hotspot=createDB file=0]`createDB()` producer method returns an instance of `MongoDatabase` that depends on the `MongoClient`. This method injects the `MongoClient` in its parameters and passes the database name into the [hotspot=getDatabase file=0]`MongoClient.getDatabase()` method to get a `MongoDatabase` instance.
 
-The [hotspot=close file=0]`close()` dispose method is a cleanup function for the `MongoClient` that closes the connection to the `MongoDB` instance.
+The [hotspot=close file=0]`close()` dispose method is a cleanup function for the `MongoClient` that closes the connection to the MongoDB instance.
 
 // =================================================================================================
 // Implementing CRUD operations
 // =================================================================================================
 
 == Performing CRUD operations
 
-You are going to implement the basic `CRUD` operations, which are `create`, `read`, `update`, and `delete`. The [hotspot=mongoImports1]`com.mongodb.client` and [hotspot=mongoImports2]`com.mongodb.client.result` packages are used to help implement these `CRUD` operations for the microservice. For more information about these packages, see the https://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/client/package-summary.html[com.mongodb.client^] and https://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/client/result/package-summary.html[com.mongodb.client.result^] Javadoc. For more information about creating a RESTful service with JAX-RS, JSON-B, and Open Liberty, see the https://openliberty.io/guides/rest-intro.html[Creating a RESTful web serivce^] guide.
+You are going to implement the basic `CRUD` operations, which are create, read, update, and delete. The [hotspot=mongoImports1]`com.mongodb.client` and [hotspot=mongoImports2]`com.mongodb.client.result` packages are used to help implement these `CRUD` operations for the microservice. For more information about these packages, see the https://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/client/package-summary.html[com.mongodb.client^] and https://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/client/result/package-summary.html[com.mongodb.client.result^] Javadoc. For more information about creating a RESTful service with JAX-RS, JSON-B, and Open Liberty, see the https://openliberty.io/guides/rest-intro.html[Creating a RESTful web serivce^] guide.
```

Comment:
Should break up paragraph into multiple lines

Reply:
Will check with ID